kuma/wiki/jinja2/wiki/document.html

```diff
@@ -262,14 +262,22 @@
 
               {% if document.current_revision %}
                 {% if document.current_revision.needs_technical_review %}
-                  <div class="warning warning-review"><p>{% trans %}
-                    This article is in need of a technical review.
-                  {% endtrans %}</p></div>
+                  <div class="warning warning-review">
+                    <p>
+                      {% trans url="https://developer.mozilla.org/docs/MDN/Contribute/HowTo/Do_a_technical_review" %}
+                        This article needs a technical review. <a href="{{ url }}">How you can help.</a>
+                      {% endtrans %}
+                    </p>
+                  </div>
                 {% endif %}
                 {% if document.current_revision.needs_editorial_review %}
-                  <div class="warning warning-review"><p>{% trans %}
-                    This article is in need of an editorial review.
-                  {% endtrans %}</p></div>
+                  <div class="warning warning-review">
+                    <p>
+                      {% trans url="https://developer.mozilla.org/docs/MDN/Contribute/Howto/Do_an_editorial_review" %}
+                        This artible needs an editorial review. <a href="{{ url }}">How you can help.</a>
```

Comment:
From: Stephanie Hobson <notifications@github.com>
 To: mozilla/kuma <kuma@noreply.github.com> 
 Sent: Friday, March 4, 2016 8:08 AM
 Subject: Re: [kuma] Fix bug 1183337 - changes wording and add link to howtos in the banner (#3765)
   
In kuma/wiki/jinja2/wiki/document.html:>                  {% endif %}
>                  {% if document.current_revision.needs_editorial_review %}
> -                  <div class="warning warning-review"><p>{% trans %}
> -                    This article is in need of an editorial review.
> -                  {% endtrans %}</p></div>
> +                  <div class="warning warning-review">
> +                    <p>
> +                      {% trans url="https://developer.mozilla.org/docs/MDN/Contribute/Howto/Do_an_editorial_review" %}
> +                        This artible needs an editorial review. <a href="{{ url }}">How you can help.</a>
Typo for "article".—
Reply to this email directly or view it on GitHub.